Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1762,6 +1820,11 @@ <h2>
     Ihnen bereitgestellten Informationen zu Ihrer Identität in der Regel nicht
     erfüllt werden können.
 </p>
+<p>
+    Wenn anlässlich der Aktualisierung eines digitalen COVID-Zertifikats der Hash-Wert der
+    elektronischen Signatur vorübergehend gespeichert wird, ermöglicht es dieser dem RKI nicht, die
```

Comment:
"vorübergehend" is a bit misleading when it's actually 365 days. Yes, one year is also "temporary" in a way, but the concrete storage period would be better in this place.
If the legal department has some capacities to have a look into it, maybe they can agree to state "365 days" here also.